gettsim/social_insurance_contributions/eink_grenzen.py

```diff
@@ -124,12 +125,59 @@ def midi_job_faktor_f(
     return out
 
 
+@add_rounding_spec(params_key="soz_vers_beitr")
+def midi_job_faktor_f_ab_2022(
+    soz_vers_beitr_params: dict,
+) -> float:
+    """Faktor F which is needed for the calculation of Bemessungsentgelt
+    (beitragspflichtige Einnahme) of midi jobs. It is calculated as the ratio of 28 %
```

Comment:
Describe where 28% comes from (also in previous function). If 28% did not change, we can still mention that number in brackets.

In the docstring or a comment in the code, it should become clear why we change from using `ges_krankenv_beitr_satz` to `soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["allgemein"]`. Couldnt we stick to `ges_krankenv_beitr_satz` and `_ges_krankenv_beitr_satz_arbeitg`?